concreate/generator.py

```diff
@@ -19,11 +19,13 @@
 
 class Generator(object):
 
-    def __init__(self, descriptor_path, target, overrides):
-        self.descriptor = Descriptor(descriptor_path, 'image').process()
-        self.target = target
+    def __init__(self, args):
```

Comment:
I don't like this change - it hides what Generator constructor needs to be passed in, who know what part of args it takes?

Reply:
The reasoning was to not endlessly extend the constructor, but I can revert it.

Reply:
see comments bellow about build(), maybe you don't need tags in constructor at all.